src/hooks/useBrowserWarning.js

```diff
@@ -0,0 +1,14 @@
+import { useEffect, useState } from 'react';
+import isChrome from '../util/isChrome';
+
+const useBrowserWarning = () => {
+  const [shouldDisplay, setShouldDisplay] = useState(false);
+  useEffect(() => {
+    if (!isChrome) {
+      setShouldDisplay(true);
+    }
+  }, [isChrome]);
+  return [shouldDisplay, () => setShouldDisplay(false)];
+};
```

Comment:
Looks good overall. Can you explain this part to me? Why can't you use the isChrome logic directly on the modal with a simple close modal button?

Reply:
This logic ensures the modal will only display once while the app is open. Refresh/opening a new tab will make the message appear again on the home page